plugins/wake/README.md

```diff
@@ -0,0 +1,57 @@
+# wakeonlan
+
+This plugin provides a wrapper around the "wakeonlan" tool available from most
+distributions' package repositories, or from [the following website](https://github.com/jpoliv/wakeonlan).
+
+To use it, add `wake` to the plugins array in your bashrc file:
+
+```bash
+plugins=(... wake)
+```
+
+## Usage
+
+In order to use this wrapper, create the `~/.wakeonlan` directory, and place in
+that directory one file for each device you would like to be able to wake. Give
+the file a name that describes the device, such as its hostname. Each file
+should contain a line with the mac address of the target device and the network
+broadcast address.
+
+For instance, there might be a file `~/.wakeonlan/server` with the following
+contents:
+
+```
+00:11:22:33:44:55:66 192.168.0.255
+```
+
+To wake that device, use the following command:
+
+```console
+wake server
+```
```

Comment:
When `console` is specified as a language, the text not followed by `$` or `#` is considered to be outputs of commands. `wake server` is the command in the present case, so it should be prefixed by `$` or `#`. Or the language specification should be `sh` or `bash` instead of `console`.